src/test/java/duke/TaskArrayListTest.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+import duke.tasks.Todo;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class TaskArrayListTest {
+
+
+    @Test
+    void addTask_increaseSize() {
+        TaskArrayList taskList = new TaskArrayList();
+        assertEquals( 0,taskList.size());
```

Comment:
I believe there should be a space after the comma, and also perhaps you could delete the extra whitespace in front of the "0" 😄